webview-ui/src/i18n/locales/en/settings.json

```diff
@@ -623,51 +623,51 @@
+			"label": "Use Inline Terminal (recommended)",
+			"description": "Run commands in a background process and stream output in chat (outside the VS Code terminal). Avoids shell profiles and shell integration, improving reliability and preventing 'Shell Integration Unavailable' errors. Recommended. <0>Learn more</0>"
 		},
 		"commandDelay": {
 			"label": "Terminal command delay",
-			"description": "Delay in milliseconds to add after command execution. The default setting of 0 disables the delay completely. This can help ensure command output is fully captured in terminals with timing issues. In most terminals it is implemented by setting `PROMPT_COMMAND='sleep N'` and Powershell appends `start-sleep` to the end of each command. Originally was workaround for VSCode bug#237208 and may not be needed. <0>Learn more</0>"
+			"description": "Adds a small delay after each command to help VS Code terminals flush all output. Set to 0 to disable. Workaround for terminal timing issues; usually not needed. <0>Learn more</0>"
 		},
 		"compressProgressBar": {
 			"label": "Compress progress bar output",
-			"description": "When enabled, processes terminal output with carriage returns (\\r) to simulate how a real terminal would display content. This removes intermediate progress bar states, retaining only the final state, which conserves context space for more relevant information. <0>Learn more</0>"
+			"description": "Recommended. Processes carriage returns (\\r) and backspaces (\\b) so progress bars and spinners render like a real terminal and only the final state is kept. Saves tokens; disable only when you need to inspect intermediate updates. <0>Learn more</0>"
 		},
 		"powershellCounter": {
 			"label": "Enable PowerShell counter workaround",
-			"description": "When enabled, adds a counter to PowerShell commands to ensure proper command execution. This helps with PowerShell terminals that might have issues with command output capture. <0>Learn more</0>"
+			"description": "Adds a small counter to PowerShell commands to work around missing output or repeated-command issues on some setups. <0>Learn more</0>"
 		},
 		"zshClearEolMark": {
 			"label": "Clear ZSH EOL mark",
-			"description": "When enabled, clears the ZSH end-of-line mark by setting PROMPT_EOL_MARK=''. This prevents issues with command output interpretation when output ends with special characters like '%'. <0>Learn more</0>"
+			"description": "Clears ZSH's end-of-line mark (%) so output is parsed correctly. <0>Learn more</0>"
 		},
 		"zshOhMy": {
 			"label": "Enable Oh My Zsh integration",
-			"description": "When enabled, sets ITERM_SHELL_INTEGRATION_INSTALLED=Yes to enable Oh My Zsh shell integration features. Applying this setting might require restarting the IDE. <0>Learn more</0>"
+			"description": "Enables Oh My Zsh shell integration detection. May require restarting the IDE. <0>Learn more</0>"
```

Comment:
Power users that are interested in this feature need to know what the `environment variable` is, so they understand that it is being set. Otherwise, they may be confused about behavior that could conflict with their existing implementation.